lib/Runtime/Library/JavascriptFunction.cpp

```diff
@@ -621,6 +618,11 @@ namespace Js
         }
 #endif
 
+// xplat-todo: This __try/__except catches AV and resumes from out of bound TypedArray access.
```

Comment:
I think for now, the plan should be to disable this feature on linux. I think you should change JavascriptArrayBuffer's constructor to use malloc on linux always, and also assert that JavascriptArrayBuffer::AllocWrapper is never used on linux. Also, in TypedArray::NewInstance, compile assert virtualAllocated is false on linux.